packages/website/tsconfig.json

```diff
@@ -1,6 +1,12 @@
 {
   "extends": "astro/tsconfigs/strictest",
   "compilerOptions": {
-    "types": [ "astro/client" ]
+    "types": ["astro/client"],
+    "baseUrl": ".",
+    "paths": {
+      "$layouts/*": ["src/layouts/*"],
```

Comment:
`$` seems unusual. Any reason not use `@` which I've seen much more often?

Reply:
no reason. i've seen `$` more so it's just a stylistic thing. I can change it. Also one of the justifications I know is using `@` can be confused/conflict with npm packages

Reply:
Fair enough. [Astro docs](https://docs.astro.build/en/guides/aliases/) use `@` so I'd stick to that instead